core/src/main/scala/org/dbpedia/extraction/ontology/OntologyDatatypes.scala

```diff
@@ -458,9 +458,9 @@ object OntologyDatatypes
         types :::= builder.build
 
         builder.addDimension("Speed");
-        builder.addUnit(new StandardUnitDatatype("kilometrePerHour", Set("km/h","kmh","kilometre per hour",/*el*/ "χιλιόμετρα ανά ώρα")));
+        builder.addUnit(new StandardUnitDatatype("kilometrePerHour", Set("km/h","kmh","kilometre per hour",/*el*/ "χιλιόμετρα ανά ώρα", /*am*/ "ኪሎሜትር በሰዓት")));
```

Comment:
kilometer vs kilometre, and plural vs singular vary regionally (not just by language). This might need to be more complicated...
```suggestion
        builder.addUnit(new StandardUnitDatatype("kilometersPerHour", Set("km/h","kmh","kilometers per hour",/*el*/ "χιλιόμετρα ανά ώρα", /*am*/ "ኪሎሜትር በሰዓት")));
```

Reply:
we could add both versions in the list to accommodate both variations